infra/helper.py

```diff
@@ -680,9 +680,15 @@ def _get_fuzz_targets(project_name):
   for name in os.listdir(_get_out_dir(project_name)):
     if name.startswith('afl-'):
       continue
+    if name.startswith('jazzer_'):
+      continue
+    if name == 'llvm-symbolizer':
+      continue
 
     path = os.path.join(_get_out_dir(project_name), name)
-    if os.path.isfile(path) and os.access(path, os.X_OK):
+    # Python and JVM fuzz targets are only executable for the root user, so
+    # we can't use os.access.
+    if os.path.isfile(path) and (os.stat(path).st_mode & 0o111):
```

Comment:
Uch. Maybe we should change things during the build to make them executable for everyone?

Reply:
Can I just submit a PR that changes this across the docs and all projects? 

I remember copying that bit from the Python docs when I set up Jazzer in OSS-Fuzz without questioning it.

Reply:
I think that makes sense.